lighthouse-core/lib/third-party-web.js

```diff
@@ -8,19 +8,22 @@
 const thirdPartyWeb = require('third-party-web/httparchive-nostats-subset');
 
 /** @typedef {import("third-party-web").IEntity} ThirdPartyEntity */
+/** @typedef {import("third-party-web").IProduct} ThirdPartyProduct */
 
 /**
- * `third-party-web` throws when the passed in string doesn't appear to have any domain whatsoever.
- * We pass in some not-so-url-like things, so make the dependent-code simpler by making this call safe.
  * @param {string} url
  * @return {ThirdPartyEntity|undefined}
  */
 function getEntity(url) {
-  try {
```

Comment:
Does this not throw an error anymore? cc @patrickhulce

Reply:
I don't think it throws anymore https://github.com/GoogleChrome/lighthouse/pull/11290#discussion_r498364311

Reply:
yep at least not in ways we expect :)